deployment/ansible/roles/raster-foundry.base/defaults/main.yml

```diff
@@ -10,7 +10,9 @@ envdir_config:
   RF_CACHE_HOST: "{{ redis_host }}"
   RF_CACHE_PORT: "{{ redis_port }}"
   AWS_PROFILE: "{{ aws_profile }}"
-  RF_AWS_BUCKET: "{{ aws_bucket }}"
```

Comment:
This variable is used but has not been renamed: https://github.com/lewfish/raster-foundry/blob/b7113316e3c7c14ba8d9046420061d76f7143665/src/rf/rf/settings/base.py#L282